paddle/framework/data_transform_test.cc

```diff
@@ -127,7 +127,9 @@ TEST(DataTransform, Layout) {
   }
 
   Tensor dst = out.Get<Tensor>();
-  EXPECT_TRUE(dst.layout() != src->layout());
+
+  EXPECT_TRUE(dst.layout() == DataLayout::kNCHW);
+  EXPECT_TRUE(dst.dims() == make_ddim({2, 2, 3, 1}));
```

Comment:
The unit test of `NCHW -> NHWC` is also need.

Reply:
done.